File: FizzBuzz/fizzbuzz.py
class FizzBuzz:
    def __init__(self):
        self.__fizz = 3
        self.__buzz = 5

    def __isFizz(self, num):
        return num % self.__fizz == 0
    
    def __isBuzz(self, num):
        return num % self.__buzz == 0

    def fizzbuzz(self, num):
        for i in range(1, num + 1, 1):
            if self.__isFizz(i) & self.__isBuzz(i):
                print("FizzBuzz")
            elif self.__isFizz(i):
                print("Fizz")
            elif self.__isBuzz(i):
                print("Buzz")
            else:
                print(i)

fz = FizzBuzz()
fz.fizzbuzz(100)

File: FizzBuzz/FizzBuzz.php
<html>
    <head>
        <title> kanagayo test </title>
    </head>
    <body>
        <?php
            define("FIZZ", 3);
            define("BUZZ", 5);
            class FizzBuzz {
                private function is_fizz($num) {
                    return ($num % FIZZ == 0);
                }
                private function is_buzz($num) {
                    return ($num % BUZZ == 0);
                }
                public function fizz_buzz($num) {
                    for ($i = 1; $i <= $num; $i++) {
                        if ($this->is_fizz($i) && $this->is_buzz($i)) {
                            echo "FizzBuzz";
                        } elseif ($this->is_fizz($i)) {
                            echo "Fizz";
                        } elseif ($this->is_buzz($i)) {
                            echo "Buzz";
                        } else {
                            echo $i;
                        }
                        echo "<br>";
                    }    
                }    
            }
            $fz = new FizzBuzz();
            $fz->fizz_buzz(100);
        ?>
    <body>
<html>